### DISCUSSION TITLE
info on failed solves

### DISCUSSION THREAD

**Discussion by jessecarterMOOSE** [idaholab/moose#24318] [Q&A General] [answered]
Is there any info available, or can any be saved, about _failed_ solves during a run? I'm hoping for something systematic that doesn't involve grepping through stdout.

**Comment by GiudGiud** [comment]
There s the troubleshooting failed solve page. 

 I m revamping rn it I ll post a link to the page in my PR

**Reply by GiudGiud** [reply]
Current page
https://mooseframework.inl.gov/releases/moose/v1.0.0/application_usage/failed_solves.html
PR page isnt generated yet

**Comment by jessecarterMOOSE** [comment]
So far cutting the time step on a failed solve has been sufficient for me
to get to the end of the run. It's detecting if and when those cutbacks
occurred that I'm interested in.

**Reply by GiudGiud** [reply, marked as answer]
oh my bad

there s this
https://mooseframework.inl.gov/source/postprocessors/NumFailedTimeSteps.html

**Reply by GiudGiud** [reply]
this will give you 'if' and 'how often'. It wont give you 'when'. But with a ParsedPostpcoessor and a TimePostprocessor you could detect 'the first time when' or 'the last time when' or the 'number of times when' pretty easily

**Reply by jessecarterMOOSE** [reply]
That should do the trick. Thanks. Wasn't aware of that postprocessor (there are so many!).
